Week1/homework/app.js

```diff
@@ -1,11 +1,114 @@
 'use strict';
 
-{
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
-  ];
-
-  // Replace with your own code
-  console.log(bookTitles);
+// never-used so I commented out
+// const bookTitles = [
+//   // Replace with your own book titles
+//   'The_Lord_the_Rings',
+//   'The_Little_Prince',
+//   'Harry_Potter_Philosopher_Stone',
+//   'The_Hobbit',
+//   'And_Then_There_Were_None',
+//   'Dream_the_Red_Chamber',
+//   'The_Lion_the_Witch_the_Wardrobe',
+//   'She_A_History_Adventure',
+//   'The_Adventures_Pinocchio',
+//   'Vardi_Wala_Gunda',
+// ];
+
+const bookDetails = {
+  The_Lord_the_Rings: {
```

Comment:
I'd suggest using all lowercase characters for your key names here - that way it'd mean one less thing to worry about when deciding uppercase/lower for words like `the`.